.circleci/config.yml

```diff
@@ -317,15 +345,15 @@ jobs:
       - save_cache:
           paths:
             - ~/venv
-          key: cache-key-cpu-py39-180-{{ checksum "setup.py"}}-{{ checksum "requirements-test.txt"}}
+          key: cache-key-cpu-py39-180-3-{{ checksum "setup.py"}}-{{ checksum "requirements-test.txt"}}
 
       - <<: *install_repo_cpu
 
       - <<: *run_isort
       - <<: *run_black
       - <<: *run_mypy
       - <<: *run_flake8
-      - run_unittests
+      - <<: *run_unittests
```

Comment:
mind elaborating on what that does ?

Reply:
Yes, of course. The old line was calling a function, which was doing all unit test when not given an option. Now, since the function take a test file list, it doesn't run all tests by default. I just reverted to use an alias that runs all possible tests on CPU. Will add a comment to the function and the alias.

Reply:
thanks !